RecoBTag/DeepFlavour/interface/btag_features_converter.h

```diff
@@ -0,0 +1,44 @@
+#ifndef RecoBTag_DeepFlavour_btag_features_converter_h
+#define RecoBTag_DeepFlavour_btag_features_converter_h
+
+namespace deep {
+
+  static constexpr std::size_t max_jetNSelectedTracks =100;
+
+  template <typename TagInfoVarsType, typename TagInfoFeaturesType>
+  void btag_features_converter( const TagInfoVarsType & tag_info_vars, TagInfoFeaturesType & tag_info_features) {
```

Comment:
this could as well be a class with only this static method, but also with the max_jetNSelectedTracks
so that this const has a more clear provenance

Reply:
Good idea, I am making all the converters classes with static methods

Reply:
changed in https://github.com/cms-sw/cmssw/pull/19893/commits/c58a7db3f5b57532c429a7f2f102b462285b57a4